marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/builtins/steps/mapping/map-multiple-entities/test-data/entities/Order.entity.json

```diff
@@ -0,0 +1,48 @@
+{
+  "info": {
+    "title": "Order",
+    "version": "0.0.1",
+    "baseUri": "http://marklogic.com/example/"
+  },
+  "definitions": {
+    "Order": {
+      "primaryKey": "orderId",
+      "properties": {
+        "orderId": {
+          "datatype": "integer"
+        },
+        "orderedBy": {
+          "datatype": "integer",
+          "relatedEntityType": "http://example.org/Customer-0.0.1/Customer",
+          "joinPropertyName": "customerId"
+        },
+        "deliveredTo": {
+          "datatype": "integer"
+        },
+        "lineItems": {
+          "datatype": "array",
+          "items": {
+            "$ref": "#/definitions/LineItem"
+          }
+        }
+      }
+    },
+    "LineItem": {
+      "required": [],
```

Comment:
To keep these tests as simple as possible (and thus easier to understand), remove these empty arrays, they're just adding noise.